care/facility/api/serializers/post_covid_data.py

```diff
@@ -0,0 +1,176 @@
+import enum
+from rest_framework import serializers
+from config.serializers import ChoiceField
+from care.facility.api.serializers import TIMESTAMP_FIELDS
+from care.facility.models.post_covid_data import PostCovidData
+
+
+# choices for mode_of_transmission attribute of anticoagulants_drugs field
+class ModeOfTransmission(enum.Enum):
+    IV = "IV"
+    ORAL = "Oral"
+
+
+ModeOfTransmissionChoices = [(e.value, e.name) for e in ModeOfTransmission]
+
+
+# Validator serializer for vitals_at_admission
+class VitalsAtAdmissionSerializer(serializers.Serializer):
+    pr = serializers.IntegerField(allow_null=True)
```

Comment:
Would it be better to come up with full names? BP is a common term, but names like `pr`, `rr` are not common terms and we cannot expect the next developers to understand the meaning.